tests/manual-test-cases/Group18-VIC-UI/18-2-VIC-UI-Uninstaller.robot

```diff
@@ -22,10 +22,14 @@ Suite Teardown  Uninstall VCH
 
 *** Test Cases ***
 Ensure Vicui Is Installed Before Testing
```

Comment:
Will move `Ensure Vicui Is Installed Before Testing` and a similar one in 18-1 to suite setup. Since the actual UI plugin test happens in 18-3, all that matters for 18-1 and 18-2 is whether or not the extension is registered with VC.